### DISCUSSION TITLE
Request for JetBrains Fleet

### DISCUSSION THREAD

**Discussion by Kwadz** [Homebrew/discussions#3796] [Casks]
[Fleet](https://www.jetbrains.com/fleet/), a new IDE from Jetbrains, has been released. It would be nice to have it via Homebrew.

**Comment by SMillerDev** [comment]
This should be made as a pull request, not a discussion

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Adding-Software-to-Homebrew

**Reply by Kwadz** [reply]
I tried to create a new [fleet.rb](https://github.com/Kwadz/homebrew-cask/blob/add-jetbrains-fleet/Casks/fleet.rb) file, by taking [phpstorm.rb](https://github.com/Kwadz/homebrew-cask/blob/add-jetbrains-fleet/Casks/phpstorm.rb) as example, but I'm not currently able to translate everything for Fleet.

**Reply by p-linnane** [reply]
@Kwadz Please open a pull request and I will help you work through it.

**Reply by Kwadz** [reply]
Thanks @p-linnane, here is the pull request: https://github.com/Homebrew/homebrew-cask/pull/133700.
